examples/shader-coverage-bvh-traversal/main.cpp

```
@@ -0,0 +1,649 @@
+    auto nodes = buildBVH(tris);
+    std::cout << "BVH: " << nodes.size() << " nodes\n";
+    auto rays = generateRays(kRayGridDim);
+    std::cout << "rays: " << rays.size() << " (" << kRayGridDim << "x" << kRayGridDim << ")\n";
+
+    Globals globalsData = {};
+    globalsData.rayCount = (uint32_t)rays.size();
+    globalsData.triCount = (uint32_t)tris.size();
+    globalsData.nodeCount = (uint32_t)nodes.size();
+
+    vkdemo::Context ctx;
+    ctx.init();
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)